### DISCUSSION TITLE
List of Compatible Power Supplies/chargers for Pinecil

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1265] [Ideas]
## Compatible Power Supplies and cables
Add comment to this thread if you find a compatible PSU that is verified to work on Pinecil with no strange reboots or other undesirable behavior. Identify if USB-C or DC type, all relevant voltage/amps/watts, if you know whether the device is UL or ETL listed, the current price and note if there is a special sale/coupon presently. 

For cables, the ideal would be silicone coated heat-resistant type that is a minimum of PD60W rated or higher and Silicone DC5525 type cable. If you find additional chargers just edit your original post to keep number of posts more organized. Stock goes in & out for all vendors, sometimes you might see currently unavailable for items.

**Try to follow this format for listing, and notes so people get any relevant details**
Some of these may go out of stock sometimes

**USB-C Charger**
$11, 65W, output PD20V/3.25Amp, 3-port, ETL listed [USA link](https://smile.amazon.com/gp/product/B09N3PG999)

**Power Banks/Batteries**

**Cables**
$10, Usb-C, PD60W, 2-Pack silicone cables [USA link](https://smile.amazon.com/gp/product/B096XWD2HG)

=========EPR below =================
Below is for newest V2 Pinecil sold after August 2,2022 from official pine64.com (ships from china) or  pine64eu.com  (ships from EU).
V2 is made to support newest EPR PD3.1 28V  USB-C. You can continue to use PD65w/20V chargers, the EPR chargers will just make it heat and re-heat a lot faster. It takes about 15s to hit temperature from cold tip with a PD65w-20V charger and ~4s with new EPR 28V charger.
With the new designed shorter tip of 6.2 ohm resistance, a 140W EPR charger will be about 126W to Pinecil.  Even on a 20V charger using the new short tip, Ralim says,  "at the moment we can peak to 3.146A @ 20V on the 6.2 ohm tip."
For reference the standard TS100 tip = 8ohm.

**EPR PD 3.1 Chargers**:  only two on the market currently but more will be coming every month as this is a new standard
- Apple 140w  usb-c  $99.00  
- Anker 140w usb-c $99.99  (not tested yet)  717 140W- 28V

**EPR 240W Cable** - for new V2 Pinecil (if you want to take advantage of 28V)
This cable linked is both 240w & high data transfer 40Gbps  TB3/TB4  (It's an all-in-one cable so it cost more since it also does highest data transfer that is  USB-IF certified). I use this one, works great and has a nice reinforced neck. Technically a power only EPR would be enough, but I wanted to have one cable that did everything data too.

current coupon brings it to $35.99, click link in this special
[Amazon-USA](https://www.amazon.com/gp/mpc/A1IOHY7DSV21KW?maas=maas_adg_85198654B51D1DA9F839087F88D56E8E_afap_abs)

If you are in EU  or can not use Amazon-USA, use below link
promo coupon code:  _PINE64ONLY_
In check out cart, click where it says promo code and a box appears.
[EU/other country Coupon for EPR cable](https://www.inter9ers.com/store?store-page=INTER9ERS-i9-All-In-One-Super-Cable-4ft-EPR-240W-40Gbps-USB4-p464220399)

**Comment by cronald-sys** [comment]
**USB-C Charger**

- $29, 65w, output PD20v/3.25A, Single port, UL Listed, Extremely compact [USA Link](https://www.amazon.com/Eggtronic-Sirius-Universal-Charger-Adapter/dp/B07XYW7YJC)


- *wholesale listing, not sure what retail price is, or if it can be found anywhere* 65w, PD20v/3.25A, QC 20v/2A, 2x USB C, 1x USB A [Wholesale Link](https://www.globalsources.com/AC-DC-adapter/65W-GaN-charger-1182223485p.htm)

**Power Banks/Batteries**

- $129.99, 60w, output PD20v/3A, QC12v/1.5A [USA Link](https://www.ravpower.com/products/rp-pb201-pd-60w-20000mah-portable-charger)

- $199, 100w, output PD20v/5A, QC12v/1.5A. DC3.3-25.2V/3A [USA Link](https://www.shargeek.com/products/storm2)





**Not currently working**

- $42, 140w EPR, output PD 25v/5A, QC 12v/1.5A USB C & A [Aliexpress link](https://www.aliexpress.us/item/3256804231324937.html?spm=a2g0o.order_list.0.0.53d218029e9DOX&gatewayAdapt=glo2usa&_randl_shipto=US) - Note: Pinecil V2 gets stuck in a bootloop when running off this power supply using a 240w cable. If a 60w cable is used instead, the iron operates as expected at 20v

**Comment by discip** [comment]
**[Anker Nano II 45W](https://www.anker.com/eu-de/products/a2664?variant=41566778851491)**

**Comment by ghost** [comment]
$19 EFAITHFIX 65W 3 ports https://www.amazon.com/gp/product/B09SQ5Q5HC
$35 Baseus 65W 3 ports https://www.amazon.com/gp/product/B083J229RJ
$72 Baseus 65W 30,000 mAh beast of a power bank https://www.amazon.com/gp/product/B08JV4W4NY

**Comment by River-Mochi** [comment]
**$21 with current coupon HTC PD65W-20V**, 3 ports,  two are USB-C. white color, can only use one port at a time if you want 65W [Amazon-USA](https://www.amazon.com/HTC-Charger-Compact-Foldable-MacBook/dp/B09J83YZ8M) normally on sale for $23.

**$18.39 w/ current $5 off coupon HTC PD65W-20V**, 3 Ports, black. normally $23 [Amazon-USA](https://www.amazon.com/Charger-HTC-PowerIQ-Foldable-Charging/dp/B09F6HF28L)

HTC brand makes phones and some gaming virtual equipment.

Don't forget most of these chargers, you can only plug One item in to get the ull 65W or it will split it up between them. sometimes only one port does the 65W, so read details on it.

**Comment by ghost** [comment]
This Rocoren 140W charger I bought for $59.99 (had $10 coupon) https://www.amazon.com/gp/product/B0B79JGV2F  And this Club3d 240W EPR cable that was $9.47 https://www.ebay.com/itm/255741699229 seem to work fine with Pinecil V2  The Club3d EPR cable can also be found for a bit more $$$ on Amazon.

**Reply by River-Mochi** [reply]
You will need to use the 2.19  Beta for V2  for   many of the EPR  chargers.  

did your Rocoren  EPR work with V2,  2.18  firmware Stock or did you need to upgrade to Beta 2.19? 
thanks

**Reply by ghost** [reply]
I needed to upgrade.  With 2.18 I was presented with just the flashy/glitchy screen.

**Comment by gamertaboo** [comment]
Lost my TS100 and saw this pinecil v2 for like $35 on amazon with next day shipping. (Love the iron btw it's amazing.) Where-ever my TS100 is the power cord is with it so I had nothing. I decided to try my Nintendo Switch AC Adapter and it works perfectly, no issues. Hope it's ok to throw in here as I couldn't find any info on it anywhere. Steam Deck adapter works as well. Worked in a pinch but I'll probably get a beefier 65w.

$29.99 USD (if you have to buy it don't) Nintendo Switch Official AC Adapter (USB Type C) - https://amzn.to/3MYOtAe
15v @ 2.6a (39w)

$24.99 - Steam Deck official AC Adapter (USB Type C) - https://www.ifixit.com/products/steam-deck-ac-adapter-us
15v @ 3a (45w) or 20v @ 2.5a (50w) (lists both of these on it)

**Reply by Depuuty** [reply]
DO NOT USE A NINTENDO SWITCH CHARGER
it will break the pinecil (https://wiki.pine64.org/wiki/Pinecil_Power_Supplies#DC_Barrel_Power)

I don’t have a Steam Deck so I’m not sure on the polarity of that charger but it could be wrong as well.

**Reply by StevenSeifried** [reply]
The Nintendo Switch USB-C power supply and the Super Nintendo power supply (with center-negative, shown in the picture on the linked site) are two complete different power supplies.

The Nintendo Switch power supply is compatible with "Limited usability" according to the Pinecil wiki

**Comment by Phixion35** [comment]
Purchased these to use with my Pinecil V2 when I first bought it at the beginning of January 2023...

$19.99 (as of 2/27/24) - [Anker USB-C to USB-C Cable, 643 Cable 100W 6ft, USB 2.0 Type C Charging Cable](https://www.amazon.com/gp/product/B093GGNRZM?th=1)

** Currently $24.99 after $6 off coupon (normally $30.99 but price fluctuates)- [Nekteck 65W USB C Charger GaN II with 6.6ft Cable, PD 3.0 PPS [USB-IF Certified] Fast Foldable Small USB C Wall Charger](https://www.amazon.com/gp/product/B09KRZFGM8)

And a great alternative silicon cable that I picked a few of and can attest to...

$6.99 (as of 2/27/24) - [Weduda - 60W USB C to USB C Cable 6ft, Soft Silicone USB 2.0 Type C Fast Charging Cable for MacBook Pro 2020, iPad Air 5, iPad Mini 6, Samsung Galaxy S21/Note 10, Huawei, Switch, PS5, Pixel](https://www.amazon.com/dp/B09QKNPFM6)  (The cable is currently white, previously and the ones I own of it are black).

**Reply by KawaiiJess** [reply]
> ** Currently $24.99 after $6 off coupon (normally $30.99 but price fluctuates)- [Nekteck 65W USB C Charger GaN II with 6.6ft Cable, PD 3.0 PPS [USB-IF Certified] Fast Foldable Small USB C Wall Charger](https://www.amazon.com/gp/product/B09KRZFGM8)

This one says down in the "compare with similar items" that this particular charger outputs 65 volts, but that seems to be a typo.

https://m.media-amazon.com/images/I/61Z9ywQuOKL.jpg

It seems to be able to do up to 21V, 3.25A, 65W

**Reply by Phixion35** [reply]
They must have cleaned that up, I'm seeing 65W under there for it.  Hopefully no one bought it thinking it's 65 volts, lol!  In any event, I'm currently seeing that charger for $29.99 with $8 off coupon.  Can't beat a great PPS capable 65W USB-C power adaptor for $22!  This adaptor is still working very well for my Pinecil V2.
